common/formatting/layout_optimizer_test.cc

```diff
@@ -680,10 +680,11 @@ class LayoutFunctionFactoryTest : public ::testing::Test,
 
     CreateTokenInfosExternalStringBuffer(ftokens_);
 
-    auto must_wrap_pre_format_tokens = verible::iterator_range(
+    using ItType = std::vector<PreFormatToken>::iterator;
+    auto must_wrap_pre_format_tokens = verible::iterator_range<ItType>(
```

Comment:
recommend using `verible::make_range`, because functions are allowed to deduce template arguments.

Reply:
Good idea, done.